src/modlunky2/ui/trackers/runstate.py

```diff
@@ -526,6 +530,30 @@ def update_is_chain(self):
         if self.win_state is WinState.TIAMAT:
             self.fail_chain()
 
+    def update_millionaire(self, inventory: Inventory):
+        collected = inventory.money + inventory.collected_money_total
+        shop = self.get_critical_state("money_shop_total")
+        self.net_score = collected + shop
+
+        # The category requires completion, which gives at least a $100K bonus.
+        if self.net_score >= 900_000:
```

Comment:
oh man, i completely missed that this syntax was added in py3.6. Bless you for bringing it to my attention :D

Reply:
lol, I wondered whether Python had picked up this syntax I loved in Ruby. Tried and it worked 😸